docs/kb/semgrep-appsec-platform/saml-google-workspace.md

```diff
@@ -0,0 +1,26 @@
+---
+description: Learn how to set up SAML access to Semgrep AppSec Platform with Google Workspace.
+tags:
+  - Semgrep AppSec Platform
+  - SSO
+---
+
+# SAML Single Sign-on with Google Workspace
+
+The process of setting up SAML Single Sign-on for the Semgrep AppSec Platform with Google Workspace is fundamentally similar to the process for other providers. However, there are a few quirks of Google Workspace to account for.
+
+Follow these steps:
+
+1. [Set up a custom SAML app](https://support.google.com/a/answer/6087519?hl=en#zippy=%2Cstep-add-the-custom-saml-app) in Google Workspace. The default Name ID, primary email, is optimal for Semgrep AppSec Platform.
```

Comment:
```suggestion
1. [Set up a custom SAML app](https://support.google.com/a/answer/6087519?hl=en#zippy=%2Cstep-add-the-custom-saml-app) in Google Workspace. The default **Name ID** is the primary email, and this value is optimal for use with Semgrep AppSec Platform.
```

I'm not attached to this wording, but I think a bit more explanation would be helpful since I had to read this several times before understanding